pages/oss.md

```diff
@@ -37,13 +37,13 @@ Communities are critically important to the long term viability of open source p
 
-<li>Incremental Release – In instances where software cannot be developed in the open, but is otherwise appropriate for release to the public, covered agencies must develop and use an incremental release schedule and undertake all necessary steps to make the code and associated documentation available for public use. This will assist in discouraging the practice of releasing large, bulk pieces of software code, which negates many of the positive attributes of open source software.</li>
+<li>Incremental Release – In a rare instance where software cannot be developed in the open, but is otherwise appropriate for release to the public, covered agencies must develop and use an incremental release schedule and undertake all necessary steps to make the code and associated documentation available for public use. This will assist in discouraging the practice of releasing large, bulk pieces of software code, which negates many of the positive attributes of open source software.</li>
 
 <li>User Engagement – Like in the Administration's Open Data Policy, covered agencies must create a process to engage in two-way communication with users to solicit help in prioritizing the release of code and feedback on the agencies' engagement with the community. See Project Open Source for best practices and tools that can be used to implement user engagement efforts.</li>
 
-<li>Code Contributions – One of the most powerful potential benefits of OSS lies within the communities that grow around OSS projects, whereby any party can contribute new code, modify existing code, or make other suggestions to improve the software. Communities can be used to monitor changes to code, track potential errors and flaws in code, and other related activities. These kinds of contributions should be anticipated and, where appropriate, considered for integration into custom-developed Government software or associated materials.</li>
+<li>Contributions – One of the most powerful potential benefits of OSS lies within the communities that grow around OSS projects, whereby any party can contribute new code, modify existing code, or make other suggestions to improve the software. Communities can be used to monitor changes to code, track potential errors and flaws in code, and other related activities. These kinds of contributions should be anticipated and, where appropriate, considered for integration into custom-developed Government software or associated materials.</li>
```

Comment:
Once again, the diff doesn't show the change here, which is removing "Code" from "Code Contributions". As noted in the first sentence, "other suggestions to improve the software" can include things like improvements to the documentation or troubleshooting, neither of which change the code itself, but are at least as helpful.